test/TestCaseReader.cpp

```diff
@@ -37,6 +37,13 @@ TestCaseReader::TestCaseReader(string const& _filename):
 	m_unreadSettings = m_settings;
 }
 
+TestCaseReader::TestCaseReader(istringstream const& _str)
+{
+	tie(m_sources, m_lineNumber) = parseSourcesAndSettingsWithLineNumber(
+			static_cast<istream&>(const_cast<istringstream&>(_str))
```

Comment:
Here and in the next line you only need 1 level of indentation

Reply:
Like this?

```
	tie(m_sources, m_lineNumber) = parseSourcesAndSettingsWithLineNumber(
		static_cast<istream&>(const_cast<istringstream&>(_str))
	);
```

or more like this?

```
	tie(m_sources, m_lineNumber) = parseSourcesAndSettingsWithLineNumber(
	static_cast<istream&>(const_cast<istringstream&>(_str))
	);
```

Reply:
Yea the first one